client/my-sites/marketplace/marketplace-plugin-details/index.tsx

```diff
@@ -105,7 +105,12 @@ function MarketplacePluginDetails( {
 			<SidebarNavigation />
 			{ hasHardBlock &&
 				raisedBlockingMessages.map( ( message ) => (
-					<Notice status={ message.status } text={ message.message } showDismiss={ false }></Notice>
+					<Notice
+						key={ message.message }
+						status={ message.status }
+						text={ message.message }
+						showDismiss={ false }
+					></Notice>
```

Comment:
We don't need a closing `<Notice />`, we could just self-close the opening one.
```suggestion
					/>
```